interfaces/apparmor/template.go

```diff
@@ -171,29 +165,27 @@ var templateCommon = `
   # Allow User/Group lookups via common VarLink socket APIs. Applications need
   # to either consult all of them or the io.systemd.Multiplexer frontend.
   /run/systemd/userdb/ r,
-  /run/systemd/userdb/io.systemd.Multiplexer rw,
-  /run/systemd/userdb/io.systemd.DynamicUser rw,        # systemd-exec users
-  /run/systemd/userdb/io.systemd.Home rw,               # systemd-home dirs
-  /run/systemd/userdb/io.systemd.NameServiceSwitch rw,  # UNIX/glibc NSS
-  /run/systemd/userdb/io.systemd.Machine rw,            # systemd-machined
+  # /run/systemd/userdb/io.systemd.DynamicUser rw,        # systemd-exec users
+  # /run/systemd/userdb/io.systemd.Home rw,               # systemd-home dirs
+  # /run/systemd/userdb/io.systemd.NameServiceSwitch rw,  # UNIX/glibc NSS
+  # /run/systemd/userdb/io.systemd.Machine rw,            # systemd-machined
+  /run/systemd/userdb/{io.systemd.Multiplexer,io.systemd.DynamicUser,io.systemd.Home,io.systemd.NameServiceSwitch,io.systemd.Machine} rw,
```

Comment:
This could be simplified to:

```
/run/systemd/userdb/io.systemd.{Multiplexer,DynamicUser,Home,NameServiceSwitch,Machine} rw
```

Reply:
ah nice, I missed that. fixed now
thank you!